### PR TITLE
fixes #619 grammatical changes in README.md

### LINKED ISSUE #619: Grammatical errors in documentation file
**I'm submitting a ...**  (check one with "x")
- [ x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
There are certain grammatical mistakes in the documentation file

**Steps to reproduce:**
Simple grammatical changes have to be made.

### PR DESCRIPTION
**Pull request by suyash-ag09**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-
-
-

Fixes #

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-610-open-event-frontend-fossasia.surge.sh

**Comment by geekyd** [comment]
@suyash-ag09 Thank you for sending a PR. It is a good practice if you create an issue before sending a PR, so that other community members can validate it. Please read http://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia to know about all the good practices. Thanks :)

**Comment by niranjan94** [comment]
@suyash-ag09 please [squash your commits](https://davidwalsh.name/squash-commits-git), [give a clear meaning full commit message](http://chris.beams.io/posts/git-commit/) and [open an issue](https://github.com/fossasia/open-event-frontend/issues/new) and [attach it to this PR](https://github.com/blog/1506-closing-issues-via-pull-requests).

**Comment by niranjan94** [comment]
Deployment link: https://pr-610-open-event-frontend-fossasia.surge.sh

**Comment by suyash-ag09** [comment]
I want to resolve #619 with this commit.

**Comment by suyash-ag09** [comment]
@niranjan94 I've done as you told. But I'm not able to tag the issue to this PR.

**Comment by harshitagupta30** [comment]
@suyash-ag09 please give your commit a clear message. For example you can change it to
For example you can change this to: resolve grammatical errors in Readme.md 
You can use the same guide shared on gitter to see how you can update

**Comment by suyash-ag09** [comment]
I've edited my commit message. What else do I need to do @harshitagupta30 ?

**Comment by niranjan94** [comment]
@suyash-ag09 

> @niranjan94 I've done as you told. But I'm not able to tag the issue to this PR.

Just edit your PR description and add something like `Fixes #619`. 

See https://github.com/blog/1506-closing-issues-via-pull-requests

File: README.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ This project has acceptance, integration and unit tests located inside the `test
 
 **Join the development**
 
-* Before you join development, please set up the project on your local machine, run it and go through the application completely. Press on any button you can find and see where it leads to. Explore. (Don't worry ... Nothing will happen to the app or to you due to the exploring :wink: Only thing that will happen is, you'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.)
-* If you would like to work on an issue, drop in a comment at the issue. If it is already assigned to someone, but there is no sign of any work being done, please free to drop in a comment so that the issue can be assigned to you if the previous assignee has dropped it entirely.
+* Before you join development, please set up the project on your local machine, run it and go through the application completely. You can press on any button to find and see where it leads to. Explore. (Don't worry ... Nothing will happen to the app or to you due to the exploring :wink: Only thing that will happen is, you'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.)
```
Comment (niranjan94) [inline README.md]: There is a change in meaning here.

> Press on any button you can find and see where it leads to (what was already there)

Thus nudges the user to literally press on anything he/she can see on the website and see where it leads to. 

> You can press on any button to find and see where it leads to (Your change)

This tells the user that pressing on a button would help them see where it leads to. Which is pretty obvious. 

So, pleaser revert this change and let the previous phrase stay untouched.

**Comment by niranjan94** [comment]
Deployment link: https://pr-610-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-610-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-610-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-610-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-610-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-610-open-event-frontend-fossasia.surge.sh
